src/ray/gcs/redis_actor_info_accessor.cc

```diff
@@ -92,7 +92,7 @@ Status RedisActorInfoAccessor::AsyncSubscribeAll(
     const SubscribeCallback<ActorID, ActorTableData> &subscribe,
     const StatusCallback &done) {
   RAY_CHECK(subscribe != nullptr);
-  return actor_sub_executor_.AsyncSubscribe(ClientID::Nil(), subscribe, done);
+  return actor_sub_executor_.AsyncSubscribeAll(ClientID::Nil(), subscribe, done);
```

Comment:
not specific to this PR, but is client_id required for AsyncSubscribeAll?

Reply:
Then we can remove the client_id parameter from this function?